metrics/pull-requests-closed.md

```diff
@@ -1,18 +0,0 @@
-# Pull Requests Closed
```

Comment:
I am concerned about deleting metric names that we have been working with. 
I agree with this change in name because it is more general, but we only just aligned these old names with the main metrics repository; So I think there is a cascading effect ... a contract of sorts, between us and the main metrics repo that has to abided along with this deletion. 
1. Rename metric here
2. Implement markdown of renamed metric new names
3. implement a pull request into the main metrics repository simultaneously to ensure ongoing alignment. 

If people know what a thing is called and have referenced it I think we cause ourselves problems by just taking it away.

Reply:
We haven't released yet, so anything up to now is subject to be changed, i think... That's the reason why I don't understand a lot the metrics repo. But I'm happy to follow this process you say if you feel it helps. I, or somebody else, could do 2. and 3. on the metrics repo once the change is accepted here.